specification/nginx/resource-manager/readme.go.md

```diff
@@ -21,9 +21,19 @@ azure-arm: true
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2022-08
   - tag: package-2021-05-preview
 ```
 
+### Tag: package-2022-08 and go
+
+These settings apply only when `--tag=package-2022-08 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-08' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2022-08-01/$(namespace)
+```
+
```

Comment:
We prefer if you could revert the changes in this file, because the track 1 Go SDK will be deprecated very soon and we no longer release new versions for it. These new configurations are only for the track 1 Go SDK.